src/flash.rs

```diff
@@ -2,6 +2,11 @@
 
```

Comment:
Thanks @richardeoin I've fixed those changes you suggested.

I have one question regarding the stm32h750 which is also apart of rm033: Should I also disable this model from this feature? Something like:

```
#[cfg(any(feature = "rm0433"), not(feature = "stm32h750"))]
```

Reply:
Yes that would be sensible, since the implementation doesn't support the stm32h750. you'd also need to narrow the `required-features` in Cargo.toml by using a specific device flag rather than just`rm0433`